Source hamburg

michaeljokf opened the pull request:
added sourcing for Hamburg (ERDF 2007-2013 only so far)

Commit:
add source file NRW

add source file ESF 2007-2020, ERDF 2007-2020

Commit:
added source for hamburg

described the ERDF 2007-2013 for Hamburg

Commit:
added source ESF 2007-2014

link for source ESF 2007-2014 was sent from hamburg

cyberbikepunk commented:
@michaeljokf welcome and thanks for your contribution! It looks really great. There's only one thing that needs to be tweaked: if your resources do not have the same structure (which is the case in both regions), you need to write two description files, because one pipeline can only handle a coherent set of data. What you want to do is create subdirectories in the regions and put a `source.description.yaml` in each one.

@seegerseeger @annafralberts

Commit:
Merge remote-tracking branch 'refs/remotes/origin/master' into source_hamburg

Commit:
updated source info as discussed

two separate folders ESF 2007-2013, ERDF 2007-2013 with separate source
descriptions
